app/assets/javascripts/application.js

```diff
@@ -10,10 +10,3 @@
 //= require govuk_publishing_components/components/step-by-step-nav
 
 //= require_tree ./modules
-//= require set-ga-client-id-on-form
-
-var form = document.querySelector('.js-service-sign-in-form')
```

Comment:
I'm wondering if this class can be removed from the HTML now? From a quick search I think it was only for this feature.

Reply:
Of course, thanks 🤦